File: week1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
//accept the height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int j = 1; j <= height; j++)
    {
        for (int y = height - j; y > 0 ; y--)
        {
            printf(" ");
        }
        for (int x = 0; x < j; x++)
        {
            printf("#");
        }
        printf("  ");
        for (int x = 0; x < j; x++)
        {
            printf("#");
        }
        printf("\n");

    }


}

File: week6/mario.py
from cs50 import get_int
while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        for i in range(height):
            for k in range(height - i - 1):
                print(" ", end="")
            for l in range(i + 1):
                print("#", end="")
            print("  ", end="")
            for j in range(i + 1):
                print("#", end="")
            print()
        break
